utils/bazel_test.py

```diff
@@ -57,7 +57,8 @@ def test_get_times(self, process_mock, unused_time_mock, unused_get_pid_mock):
   @mock.patch.object(bazel.Bazel, '_get_heap_size')
   @mock.patch.object(bazel.Bazel, '_get_times')
   @mock.patch.object(bazel.subprocess, 'Popen')
-  def test_command(self, subprocess_mock, get_times_mock, get_heap_size_mock,
+  @mock.patch('datetime.datetime')
```

Comment:
how are the changes to this file related to the PR?

Reply:
Moved that to PR #14